ducktape/cluster/remoteaccount.py

```diff
@@ -62,6 +63,8 @@ def _can_ping_url(self, url, headers):
     def ssh_command(self, cmd):
         if self.local:
             return cmd
+        if self.sudo_user is not None:
```

Comment:
Do we always want to apply the sudo user? Not sure I understand all the use cases, but a lot of ssh commands don't need or want sudo. If we include a `sudo_user`, wouldn't it be better to have the caller opt in (and replace all the SSH commands prefixed with `sudo` with regular calls that have a flag like `sudo=True` or `sudo="username"` or something like that)?